modules/ROOT/pages/vpn-risk.adoc

```diff
@@ -0,0 +1,52 @@
+= Anypoint VPN risk
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-active.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub"]
+image:logo-hybrid-disabled.png[xref="deployment-strategies.adoc#hybrid-deployments", title="Hybrid"]
+image:logo-server-disabled.png[xref="deployment-strategies.adoc#anypoint-platform-pce-deployments", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[xref="deployment-strategies.adoc#anypoint-runtime-fabric-deployments", title="Runtime Fabric"]
+
+Every VPN provisioned in Mulesoft has a risk associated depending on the state of the tunnels of the VPN.
+
+It's important to take into account the risk of each VPN because regularly VPNs are updated without notice.
```

Comment:
For VPNs that are in High risk, MuleSoft recommends that customers plan to deploy a xref:vpn-high-availability.adoc[high-availability VPN] solution to avoid any downtime associated with these AWS delivered security updates. Please review the xref:vpn-maintenanceadoc[recommendations and troubleshooting].